src/sdks/discovery/package.json

```diff
@@ -0,0 +1,48 @@
+{
+	"name": "@firebolt-js/discovery-sdk",
+	"version": "0.11.0-next.0",
+	"description": "The Firebolt Discovery JS SDK",
+	"main": "./dist/lib/firebolt-discovery.mjs",
+	"types": "./dist/lib/firebolt-discovery.d.ts",
+	"exports": {
+		".": "./dist/lib/firebolt-discovery.mjs"
+	},
+	"type": "module",
+	"scripts": {
+		"validate": "npx firebolt-openrpc validate --input ./dist/firebolt-discovery-open-rpc.json",
+		"sdk": "npx firebolt-openrpc sdk --input ./dist/firebolt-discovery-open-rpc.json --template ./src/js --output ./build/javascript/src",
+		"compile": "cd ../../.. && npm run compile",
+		"slice": "npx firebolt-openrpc slice -i ../../../dist/firebolt-open-rpc.json --sdk ./sdk.config.json -o ./dist/firebolt-discovery-open-rpc.json",
+		"docs": "npx firebolt-openrpc docs --input ./dist/firebolt-discovery-open-rpc.json --output build/docs/markdown --as-path",
+		"wiki": "npx firebolt-openrpc docs --input ./dist/firebolt-discovery-open-rpc.json --output build/docs/markdown",
+		"dist:notest": "npm run clean && npm run slice && npm run validate && npm run sdk && npm run docs && npm run dist:copy && echo 'Firebolt Discovery SDK /dist/ is ready.\n'",
+		"dist:copy": "npm run dist:copy:sdk && npm run dist:copy:docs",
+		"dist:copy:sdk": "mkdirp ./dist && cp -R build/javascript/src dist/lib && cp -r ./dist ../../../dist/discovery && mv ../../../dist/discovery/firebolt-discovery-open-rpc.json ../../../dist/firebolt-discovery-open-rpc.json",
```

Comment:
wonder if these mv paths will work on Windows

Reply:
from what I understand, it's basically `mkdir -p` that breaks windows... who is our resident windows user to check for us?

Reply:
@kevinshahfws : could you help us out here?